README.md

```diff
@@ -54,6 +55,37 @@ quoting mechanism.
 
 After the conversion, the bytes can be use in the field `metadata` when originating the contract.
 
+## Generating the contract storage
```

Comment:
I think this section belongs in `docs/compilation.md` instead

Reply:
Removed this and updated compilation.md